build.gradle

```diff
@@ -1,40 +1,14 @@
-// Top-level build file where you can add configuration options common to all sub-projects/modules.
-
-buildscript {
-    repositories {
-		google()
-		jcenter()
-    }
-	dependencies {
-		// NOTE - this is referencing a local variable that you must set in your gradle.properties
-		// in the top level folder.
-		//
-		// Example:
-		//
-		// project.ext {
-		//    localBuildTools = com.android.tools.build:gradle:0.14.1
-		// }
-		//
-		if (project.hasProperty('localBuildTools')) {
-			classpath project.ext.localBuildTools
```

Comment:
Don't worry about this approach. We used this in android2 repo to test the code quickly against a different version, but i doubt anybody is using it nowadays